portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,18 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+/**
+ * Adds a random movie recommendation to the page.
+ */
+function addRandomMovie() {
+  const movies =
+      ['Searching (2018)', 'The Avengers (2012)', 'Spider-Man: Homecoming (2017)', 'Your Name (2016)', 'Knives Out (2019)'];
```

Comment:
Might be good to keep line length within 80 characters limit:
https://google.github.io/styleguide/jsguide.html#formatting-column-limit

Reply:
Got it, thanks!